### PR TITLE
Addressing feedback from the AC Vote

### PR DESCRIPTION
**Pull request by wareid**
Pulling together changes to reflect feedback from the AC Vote that is currently in progress.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/PWETF/pull/371.html" title="Last updated on Mar 5, 2024, 4:34 PM UTC (3b6382e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/PWETF/371/661d901...3b6382e.html" title="Last updated on Mar 5, 2024, 4:34 PM UTC (3b6382e)">Diff</a>

### CODE REVIEW

**Comment by wareid** [comment]
I can't seem to add Mark as a reviewer, so I'm pinging here: @mnot

**Review by TzviyaSiegman — APPROVED** [review]

**Comment by TzviyaSiegman** [comment]
Thank you, Wendy. This looks great.

File: index.html [hunk 1]
```diff
@@ -92,6 +92,9 @@ <h3>
       <h2>
         Introduction
       </h2>
+      <p class="note">
+        This document was formerly known as the Code of Ethics and Professional Conduct, and references to that name may still appear in other W3C documents. If there is a reference to the Code of Ethics and Professional Coduct, or CEPC, it is referring to this document.
```
Comment (hober) [inline index.html:96]: This looks great!

**Review by hober — APPROVED** [review]
The change based on my feedback looks good, so I'll hit 'Approve'. That said, the rest of the diff is quite hard to read; I suspect there are spurious whitespace / formatting changes mixed in with the substantive ones. That's unfortunate, and makes the diff as a whole hard to review.

**Comment by mnot** [comment]
Thanks, this is an improvement.

File: index.html (hunk 1 above)
Comment (nigelmegitt) [inline index.html:96]: typo fix
```suggestion
        This document was formerly known as the Code of Ethics and Professional Conduct, and references to that name may still appear in other W3C documents. If there is a reference to the Code of Ethics and Professional Conduct, or CEPC, it is referring to this document.
```

**Comment by wareid** [comment]
> The change based on my feedback looks good, so I'll hit 'Approve'. That said, the rest of the diff is quite hard to read; I suspect there are spurious whitespace / formatting changes mixed in with the substantive ones. That's unfortunate, and makes the diff as a whole hard to review.

Thank you! I need to set up the preview link for this repo, I got a little carried away with formatting.

**Review by tantek — APPROVED** [review]
LGTM

**Review by fantasai — CHANGES_REQUESTED** [review]
-1 to all the whitespace changes. I would welcome intentional wholesale reformatting of the source code for consistency and/or introducing [semantic line breaking](https://rhodesmill.org/brandon/2012/one-sentence-per-line/); but not in this PR because it's confusing. :)

Wrt the specific changes, the Member Agreement refers to the CEPC by name (see https://www.w3.org/2023/01/Member-Agreement ) and that's a legal document, so I'm not sure it's enough to just add a historical note?

My suggestion would be to put a sentence either in the Intro or in the Status section that unambiguously says "This _is_ the W3C Code of Ethics and Professional Conduct (CEPC)". It can also say other things, like: "This is the W3C Code of Ethics and Professional Conduct (CEPC); and has been retitled Code of Conduct for [reasons]." Or whatever.

**Comment by wareid** [comment]
@chrisn Thank you for your feedback on the AC review, I have added some editorial changes in line with the feedback, would it be possible for you to review this so I can merge this PR?

**Review by TzviyaSiegman — APPROVED** [review]

**Comment by wareid** [comment]
@fantasai I have addressed your suggestion in the latest commit. I cannot do anything about whitespace at this time but I will address it in a future PR dedicated to fixing all of the inconsistencies.

**Comment by swickr** [comment]
A reviewer submitted the following comments to the Team, requesting that the Team forward them.  I think they have been adequately addressed in this PR, but documenting them here for the record:

> Section 14 should be revisited for accessibility and inclusion as the language is not easily understandable, particularly to someone who's first language is not English. A suggested alternative follows.
>
> - Wasting others time.
> - Repeating statements which have been shown to be false.
> - Re-raising questions that have already been answered.
>
> I have tested both the original text and the proposed replacement at https://app.readable.com/text/?demo. The original text is ranked as C and the replacement A.

**Comment by wareid** [comment]
@swickr Thank you for sharing the feedback, I agree that I believe the comments are addressed, but just for the benefit of the submitter, I'll provide some direct pointers that are applicable to their concerns: 

> Wasting others time.

We don't have a specific recommendation regarding "time" as it's a challenge to quantify considering our work mode (too much time in calls? Spent reading GitHub issues?), if the reviewer has a recommendation, or would like to explore the topic further, we recommend opening an issue in this repo so we can explore it for a future revision. I think it's a worthy topic to explore, I wouldn't want to attempt to force something in without discussion. 

> Repeating statements which have been shown to be false.

See [section 7](https://w3c.github.io/PWETF/index.html#misinfo), and point 4 of [section 14](https://w3c.github.io/PWETF/index.html#ad-nauseum).

> Re-raising questions that have already been answered.

See [section 14](https://w3c.github.io/PWETF/index.html#sealioning).

Re: Readability, I don't see the proposed revised text mentioned, so I'm not sure what the suggested changes are.

**Comment by fantasai** [comment]
> I cannot do anything about whitespace at this time

I'm asking to update this PR so that it doesn't include unnecessary whitespace changes so that it's easy to review what's actually changed. Not sure why that can't/shouldn't be done?

**Review by tantek — APPROVED** [review]
Updates LGTM too. Also ok with reverting the whitespace changes (presumably copy-pasting those lines from the old into additional commits on the new).

**Review by cwilso — APPROVED** [review]

**Review by chrisn — APPROVED** [review]

**Review by nigelmegitt — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -71,6 +71,9 @@
       </ul>
     </section>
     <section id="sotd">
+      <p>
+        This document is the W3C Code of Ethics and Professional Conduct (CEPC) and has been retitled Code of Conduct to reflect its scope and content. If there is a reference to the Code of Ethics and Professional Conduct (CEPC) in a W3C document or space, it is referring to this document.
```
Comment (CkuJleToH) [inline index.html:75]: [](url)

**Pull request merged**